### PR TITLE
Added Styling to RelatedContainer, Cards, and Modal 

### PR DESCRIPTION
**Pull request by Lordnickim**
Added Styling to Cards and RelatedContainer. 
Now should Render with set Height and Less responsive. 
Modal now has some styling and fixed zdepth

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ import Modal, { ModalProvider, BaseModalBackground } from "styled-react-modal";
 const FadingBackground = styled(BaseModalBackground)`
   opacity: ${(props) => props.opacity};
   transition: all 0.3s ease-in-out;
+  z-index: 2000;
```
Comment (erikeh) [inline client/src/components/App.jsx:15]: wow you REALLY want this to stand out in front lol

**Review by erikeh — APPROVED** [review]
Would love to see all the inline css converted to styled components at some point, but excited to see a formatted container!

File: client/src/components/App.jsx (hunk 1 above)
Comment (Lordnickim) [reply client/src/components/App.jsx:15]: Lools yea.... its cuz the hardcode react-multi-carosul arrow has an zindex of 1000 :/   Lols will def look into swipper next project

File: client/src/components/RelatedAction.jsx [hunk 2]
```diff
@@ -8,17 +8,17 @@ const EmptyStar='../assets/icons8-star-24-empty.png'
...
-width: 50px;
-height: 50px;
+width: 24px;
+height: 24px;
 `
 const StyledModal=Modal.styled`
 width: 500px;
 height: 500px;
-display: absolute;
+position:absolute;
```
Comment (raphaelspies) [inline client/src/components/RelatedAction.jsx]: spaces after colons are recommended by ABNB style guide

File: client/src/components/RelatedAction.jsx [hunk 3]
```diff
@@ -87,7 +87,7 @@ export default function RelatedAction ({product, overviewProduct}) {
   }
 
   return (
-    <div>
+    <div  style={{zIndex: "10", position:"absolute"}} >
```
Comment (raphaelspies) [inline client/src/components/RelatedAction.jsx]: Space after colon

File: client/src/components/RelatedAction.jsx [hunk 4]
```diff
@@ -99,15 +99,15 @@ export default function RelatedAction ({product, overviewProduct}) {
...
           <tr >
-            <th style={{padding: "5px 8px 15px", textAlign: "left"}}>{product.name}</th>
+            <th style={{padding: "5px 8px 15px", textAlign: "left", fontFamily: 'Roboto-Bold'}}>{product.name}</th>
             <th style={{textAlign: "center"}}></th>
-            <th style={{padding: "5px 8px 15px", textAlign: "right"}}>{overviewProduct.name}</th>
+            <th style={{padding: "5px 8px 15px", textAlign: "right", fontFamily: 'Roboto-Bold'}}>{overviewProduct.name}</th>
           </tr>
           </thead>
-          <tbody>
+          <tbody style={{overflowY:"scroll", maxHeight:"430px", overflow:"auto"}}>
```
Comment (raphaelspies) [inline client/src/components/RelatedAction.jsx]: Leave spaces after colons

**Review by raphaelspies — APPROVED** [review]

**Comment by Lordnickim** [comment]
Gotcha just did a commit to reflect the spacing after the colon

**Pull request merged**